deploy/main.js

```diff
@@ -49,11 +51,15 @@ app.get('/login/callback', async (req, res, next) => {
   const { code } = req.query;
   const options = {
     code,
-    redirect_uri: migMeta.oauth.redirectUri,
+    redirect_uri: migMeta.oauth.redirectUrl,
   };
   try {
+    const proxyString = process.env['HTTPS_PROXY'] || process.env['HTTP_PROXY'] || '';
+    const httpOptions = {
+      agent: new HttpsProxyAgent(proxyString),
```

Comment:
When proxy string is not set or set to "", is the Proxy ignored? I assume when the user doesn't set it, operator always sets it as empty string.

Reply:
@pranavgaikwad Updated to bypass this setting if the proxy is not set in the env vars.